### DISCUSSION TITLE
Tour operator schema

### DISCUSSION THREAD

**Discussion by mrichter1234** [schemaorg/schemaorg#3470] [General]
As a tour operator, we always come up against the fact that, in our opinion, there is no truly optimal scheme.
Among other things, "hotel" & "product" can be used. "Hotel" for hotel offers and "product" for classic round trips.
As a tour operator, however, we do not sell products in the traditional sense, but completely organized trips. These can vary from period to period. There are also different variants (flight, bus, own journey, ...). There is certainly the trip scheme for this, but in our view this is not the best solution either.

Is it not advisable to develop a scheme optimized for tour operators or have we overlooked an optimal scheme?

**Comment by mfhepp** [comment]
Actually, schema.org has a very sophisticated mechanism for this use-case (and similar, complex bundles), thanks to [`schema:includesObject`](https://schema.org/includesObject) in combination with [`schema:TypeAndQuantityNode`](https://schema.org/TypeAndQuantityNode).

This basically allows expressing which products and services are included in a [`schema:Offer`](https://schema.org/Offer), e.g.

- two nights in a hotel
- one dinner
- a spa voucher
- transportation

This mechanism comes from the original [GoodRelations vocabulary for e-commerce](https://www.heppnetz.de/projects/goodrelations/), added to schema.org in 2012.

For examples, see e.g. 
- [Issue 2605](https://github.com/schemaorg/schemaorg/issues/2605)
- the [hotels page in schema.org](https://schema.org/docs/hotels.html)
- the [section on meals in hotel offers](https://schema.org/docs/hotels.html#meals).

My gut feeling is that  the existing elements will allow very comprehensive tour operator packages.

**Reply by mrichter1234** [reply]
Thank you for your comment. Then we are not doing it wrong.
For a group tour operator, the depth and the construct is still very difficult.

The hotel has the least difficulty for the scheme.
